v1/api/swagger/components/schemas/StoragePool.yaml

```diff
@@ -0,0 +1,23 @@
+type: object
+description: >
+  An entry describing a storage pool.
+required:
+  - ID
+  - Name
+properties:
+  ID:
+    type: string
+    format: uuid
+    description: Storage pool unique ID
+  Name:
+    type: string
+    description: User-visible name for a storage pool
+  LocationID:
+    type: string
+    format: uuid
+    description: The location ID of the data center where the storage pool exists
+  Capacity:
+    type: integer
+    format: int64
+    description: >-
+      Total capacity available (in GiB) to create new volumes in the storage pool
```

Comment:
There's a push from UX/UI team to standardize all units to "xB", not "xiB", so we should stick with GB here?

Reply:
plan is to address that when decision is made to convert all units in Project API along with the inclusion of the unit in the name of the attribute and likely API versioning since it would be a disruptive change

Reply:
Sorry for the late review. This schema is used to represent a storage pool and as such LocationID and Capacity should be required.